src/index.ts

```diff
@@ -95,6 +95,12 @@ export { NEVER } from './internal/observable/never';
 
 /* Types */
 export * from './internal/types';
+export { ConnectConfig } from './internal/operators/connect';
```

Comment:
While I think it makes sense that you put these here, it's probably better just to have them next to the operators they're with. Just to help show the relationship a little more clearly.

Reply:
@benlesh, since #6488 has been merged to master, I think it would be wise to move exported interfaces back to this file and keep them exported only here.
The first reason is to avoid duplication - we don't want to export stuff in two interfaces (also, dgeni tools don't like duplicated stuff; also duplication is hard to maintain - what if a new interface is added, would a creator of that interface know that it should be added to both files?).
The second reason, if they would remain in `/src/operators/index.ts` file: that file will most probably be removed, so we want to have all exported interfaces in one place before removal so we don't remove exported interfaces accidentally.
Please let me know your thoughts on this. Thanks.